src/providers/node/mod.rs

```diff
@@ -151,32 +169,56 @@ impl NodeProvider {
     }
 
     pub fn get_start_cmd(app: &App, env: &Environment) -> Result<Option<String>> {
+        let dlx = NodeProvider::get_package_manager_dlx_command(app);
+        let pkg_manager = NodeProvider::get_package_manager(app);
+        let executor = if pkg_manager == *"bun" { "bun" } else { "node" };
+        let package_json: PackageJson = app.read_json("package.json").unwrap_or_default();
+
         if Nx::is_nx_monorepo(app, env) {
             if let Some(nx_start_cmd) = Nx::get_nx_start_cmd(app, env)? {
                 return Ok(Some(nx_start_cmd));
             }
         }
+        if Turborepo::is_turborepo(app) {
+            let turbo_cfg = Turborepo::get_config(app)?;
+            let app_name = Turborepo::get_app_name(env);
+
+            if let Some(name) = app_name {
+                if Turborepo::has_app(
+                    app,
+                    if pkg_manager == "pnpm" {
+                        turborepo::pnpm_workspaces(app)?
+                    } else {
+                        package_json.workspaces.unwrap_or_default()
+                    },
+                    &name,
+                )? {
+                    return Ok(Some(format!(
+                        "{} --workspace {} run start",
+                        pkg_manager, name
+                    )));
+                }
+                println!("warning: app `{}` not found", name);
```

Comment:
I'm unsure about this type of warning. But I think it is okay for now

```suggestion
                println!("Warning: Turborepo app `{}` not found", name);
```